web/html/src/components/hub/AddTokenButton.tsx

```diff
@@ -86,22 +86,24 @@ export class AddTokenButton extends React.Component<Props, State> {
             title={t("Add a new access token")}
             className="btn-primary"
             items={[
-              <LinkButton
+              // eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
We do not want to introduce new instances of this. `<a href="#" ...>` is bad practice and we only have it elsewhere because we haven't had the time to refactor old instances out. A simple mental model to use is that if clicking on it navigates you somewhere, it should be a link, if it performs some other action, it should be a button element. If we need the style to be different, we can use `<button className="is-plain>` or other similar options.

Reply:
I have fixed this in another PR for the dropdowns.

Reply:
Nice, I must have missed that. In that case, all good, thanks a lot for the large amount of time you've put into this. 🙂